src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(name) {
+    const bittenAnimal = Animal.alive.find(animal => animal === name);
```

Comment:
```suggestion
  bite(targetAnimal) {
```
Actually, this part is redundant as bite method should receive the whole object as an argument, not just name.
Pay attention to the task's description:
![image](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/96409563/4501df4d-1d32-4b93-aa86-290f759773ab)